README.md

```diff
@@ -71,15 +87,18 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+words = %w[Chip Potts Cogsworth Lumière Mrs. Potts]
+str = " "
+words.each { |word| str += "#{word} " }
```

Comment:
It looks like you're trying to generate a long string of "is friends" statements here. Simpler approach would be to handle all the `puts`ing inside of the each statement. So it would look something like this...

```rb
words.each do |word|
  puts "Belle is friends with #{word}"
end
```